src/services/measurementSystem.ts

```
@@ -0,0 +1,628 @@
+  text: string;
+  position: THREE.Vector3;
+  direction: 'horizontal' | 'vertical';
+  color: string;
+  isObjectDimension: boolean;
+}
+
+export class MeasurementSystem {
+  private scene: THREE.Scene;
+  private camera: THREE.Camera;
+  private renderer: THREE.WebGLRenderer;
+  private enabled: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)